tests/test_bgp_neighbor_af.rb

```diff
@@ -552,6 +670,60 @@ def send_community(af, dbg)
 
     v = 'both'
     af.send_community = v
+    assert_equal('both', af.send_community,
+                 "Test 9. #{dbg} Failed to set '#{v}' from None")
+
+    v = af.default_send_community
+    af.send_community = af.default_send_community
+    assert_equal(v, af.send_community,
+                 "Test 10. #{dbg} Failed to set state to default")
+  end
+
+  def send_comm_xr(af, dbg)
+    v = 'both'
+    af.send_community = v
+    assert_equal(v, af.send_community,
+                 "Test 1a. #{dbg} Failed to set '#{v}' from None")
+    af.send_community = v
+    assert_equal(v, af.send_community,
+                 "Test 1b. #{dbg} Failed to set '#{v}' from None")
+    v = 'extended'
+    af.send_community = v
+    assert_equal('send-extended-community-ebgp', af.send_community,
+                 "Test 2a. #{dbg} Failed to set '#{v}' from 'both'")
+    af.send_community = v
+    assert_equal('send-extended-community-ebgp', af.send_community,
+                 "Test 2b. #{dbg} Failed to set '#{v}' from 'extended'")
+    v = 'standard'
+    af.send_community = v
+    # XR it's additive
```

Comment:
In Nexus the behavior is:
standard or extended or both

So if you have extended then do standard it replaces the extended (not additive).

In XR the behavior is
standard or extended or standard and extended.

So when you do extended after a standard, you get both configured which is equivalent of 'both'.

So I think this is correct - but not 100% sure about the nexus behavior.